### PR TITLE
fix: 有冻结行且有垂直 scrollWidth 时冻结行无法 resize

### LINKED ISSUE #1593: 🐛明细表有冻结行时冻结航的resize失效
### 🏷 Version

| Package      | Version |
| -------------- | --------- |
| @antv/s2       |          |
| @antv/s2-react |          |
| @antv/s2-vue   |          |

### Sheet Type

- [ ] PivotSheet
- [x] TableSheet
- [ ] GridAnalysisSheet
- [ ] StrategySheet

### PR DESCRIPTION
**Pull request by serializedowen**
### 👀 PR includes

✨ Feature

- [ ] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [x] Solve the issue and close #1593

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌  
image
    | ✅   
image
  |

### 🔍 Self-Check before the merge

- [x] Add or update relevant docs.
- [x] Add or update relevant demos.
- [x] Add or update test case.
- [x] Add or update relevant TypeScript definitions.

### CODE REVIEW

**Review by YardWill — APPROVED** [review]
LG

File: packages/s2-core/src/cell/table-col-cell.ts [hunk 1]
```diff
@@ -49,6 +49,28 @@ export class TableColCell extends ColCell {
       this.spreadsheet.dataSet.getFieldName(this.meta.label),
     );
   }
+  
+  protected shouldAddVerticalResizeArea() {
+    if (this.isFrozenCell()) return true
+    else return super.shouldAddVerticalResizeArea()
```
Comment (lijinke666) [inline packages/s2-core/src/cell/table-col-cell.ts:55]: 这个本地 lint 应该就报错了吧, commit -n 提上来的 ? if 和 return 不要写在一行吧, 好难看

```ts
  if (this.isFrozenCell()) {
    return true
  }
  return super.shouldAddVerticalResizeArea()
```

Comment (serializedowen) [reply packages/s2-core/src/cell/table-col-cell.ts]: 好了

**Review by lijinke666 — APPROVED** [review]

**Pull request merged**